orchestrator/graphql/resolvers/version.py

```diff
@@ -0,0 +1,21 @@
+from structlog import get_logger
+
+from orchestrator.graphql.schemas.version import VERSIONS, VersionType
+from orchestrator.graphql.types import OrchestratorInfo
+from orchestrator.graphql.utils import create_resolver_error_handler
+
+logger = get_logger(__name__)
+
+
+def resolve_version(info: OrchestratorInfo) -> VersionType | None:
```

Comment:
Could you add a unittest for this?

Reply:
I think I've added a test here now, let me know if youd like that to look different, or if that resolves the ask

Reply:
s/workflow/version/g is done as well

Reply:
great thanks!